### PR TITLE
Added "simple" navigation metrics.

### PR DESCRIPTION
**Pull request by gklopper**
The intent is to have really quick feedback for when we start lazy loading some components on the bottom of the page.

(this also deletes some live blog metrics I put in a while back and no longer use).

by using a sample of browsers (those that support sendBeacon) this will give us a near real-time view of...
1. Percentage of people going to view another Guardian page.
2. Percentage of people interacting with the page in a way that does not cause another page to load (e.g. clicking a tab).
   1. Percentage of people clicking "Related content"
   2. percentage of people clicking "more form this series"

### CODE REVIEW

File: diagnostics/app/controllers/DiagnosticsController.scala [hunk 1]
```diff
@@ -47,7 +59,10 @@ object DiagnosticsController extends Controller with Logging {
     TinyResponse.noContent()
   }
 
-  def cssOptions = Action { implicit request =>
+  def cssOptions = postOptions
+
+  private def postOptions: Action[AnyContent] = Action { implicit request =>
+    println("OPTioNSSSSSSSSSSSS")
```
Comment (gklopper) [inline diagnostics/app/controllers/DiagnosticsController.scala]: oops

File: static/src/javascripts/projects/common/modules/analytics/simple-metrics.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            if (!!navigator.sendBeacon) {
+                // sm = simple metrics
+                beacon.beaconCounts('sm-page-view');
+
+                mediator.on('module:clickstream:click', function (spec) {
+
+                    if (spec.samePage) {
+                        // people who clicked something but will stay on this page
+                        clickData.push('sm-interaction-on-same-page');
+                        beacon.beaconCounts('');
```
Comment (mattosborn) [inline static/src/javascripts/projects/common/modules/analytics/simple-metrics.js]: what's this for?

Comment (gklopper) [inline static/src/javascripts/projects/common/modules/analytics/simple-metrics.js]: yeah, that should go

File: static/src/javascripts/projects/common/modules/analytics/simple-metrics.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+], function (
+    mediator,
+    beacon
+) {
+
+    // Simple metrics that only target browsers that support sendBeacon
+
+    return {
+        init: function () {
+            var clickData = [];
```
Comment (mattosborn) [inline static/src/javascripts/projects/common/modules/analytics/simple-metrics.js]: should this be initialized / empty for every click?

**Comment by mattosborn** [comment]
:+1: aside from comments

File: static/src/javascripts/projects/common/modules/analytics/simple-metrics.js (hunk 3 above)
Comment (gklopper) [inline static/src/javascripts/projects/common/modules/analytics/simple-metrics.js]: Ah, I see what you mean, good spot

**Pull request merged**